pkg/kapp/cmd/appchange/list.go

```diff
@@ -49,14 +55,30 @@ func (o *ListOptions) Run() error {
 		return err
 	}
 
-	AppChangesTable{"App changes", changes}.Print(o.ui)
+	if o.TimeFlags.Before != "" {
+		o.TimeFlags.BeforeTime, err = time.Parse(time.RFC3339, o.TimeFlags.Before)
+		if err != nil {
+			return err
+		}
+	}
+
+	if o.TimeFlags.After != "" {
+		o.TimeFlags.AfterTime, err = time.Parse(time.RFC3339, o.TimeFlags.After)
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
I think we should verify that the After time is less than the before time if both are non-nil values?
I think today we will simply list no values if it is not as none of the records satisfy those conditions.
Maybe we should error out as a user might be doing this erroneously?

Reply:
Looks like you are willing to say we should through an error when before time is less than after time.

Here after time is less than before time but this can be a valid usecase
`kapp appchange list -a <app-name> --before 2022-01-10T02:00:25Z --after 2022-01-09T11:15:25Z`
(here user wants app change list from 09 Jan 11:15 to 10 Jan 2:00)

While when before time is less than after time then it doesn't make sense. something like - 
`kapp appchange list -a <app-name> --before 2022-01-10T02:00:25Z --after 2022-01-10T11:15:25Z`

is my understanding correct?

Reply:
Yes that's correct. Switched them up by mistake.

Reply:
Done